prow/cluster/crier_deployment.yaml

```diff
@@ -20,11 +20,17 @@ spec:
       - name: crier
         image: gcr.io/k8s-prow/crier:v20191217-e7571bd25
         args:
+        - --github-workers=1
+        - --github-endpoint=http://ghproxy
+        - --github-endpoint=https://api.github.com
         - --slack-workers=1
         - --slack-token-file=/etc/slack/token
         - --config-path=/etc/config/config.yaml
         - --job-config-path=/etc/job-config
```

Comment:
Please specify `--github-token-path` even if the automagic default happens to be correct.

Reply:
Now explicitly setting `--github-token-path`.